src/recipe.cpp

```diff
@@ -189,6 +189,15 @@ void recipe::load( JsonObject &jo, const std::string &src )
         }
     }
 
+    if( jo.has_member( "delete_flags" ) ) {
+        flags_to_delete.clear();
```

Comment:
If you make `flags_to_delete` a `std::set` (which it should be), you can load it via `flags_to_delete = jo.get_tags( "delete_flags" );`.